### ISSUE TITLE
Project still going?

### ISSUE THREAD

**Issue by SavageAUS** [alcheck/transshift#10]
Is this project still going?
If it is where can i download?

**Comment by alcheck** [comment]
Hi,

This app was rejected by apple so there is no place to download signed IPA file. To use this app you should download it and compile by yourself.

**Comment by SavageAUS** [comment]
Yeah I tried that but I’m on 10.14 and it wouldnt compile. What’s version of Xcode and macOS are supported?

**Comment by alcheck** [comment]
I didn’t test it with the latest xcode - so I think I will do some fixes to support the xcode 9 when I have a time. The app is only for iOS. Thanks for your feedback.

**Comment by lukasa1993** [comment]
what was there reason for rejecting?

**Comment by trejo08** [comment]
what's the last version compiled of the app, I got a version but I don't know if is the last version, my current version is: 1.0.0(411).

If someone have an updated version, could notify me to get the last.

Thanks

**Comment by Gelioks** [comment]
> I didn’t test it with the latest xcode - so I think I will do some fixes to support the xcode 9 when I have a time. The app is only for iOS. Thanks for your feedback.
> […](#)
> 26 авг. 2018 г., в 16:49, srscott ***@***.***> написал(а): Yeah I tried that but I’m on 10.14 and it wouldnt compile. What’s version of Xcode and macOS are supported? — You are receiving this because you commented. Reply to this email directly, view it on GitHub, or mute the thread.

Looking forward to the updated version. Many thanks in advance!

**Comment by gameman733** [comment]
> Hi, This app was rejected by apple so there is no place to download signed IPA file. To use this app you should download it and compile by yourself.
> […](#)

Is it possible to post unsigned ipa files, given that there are options out there to install those with a little more work? See https://altstore.io/ for one example, and I believe Cydia has an app for more local installs.

**Comment by salixh5** [comment]
Doesn't compile on my machine sadly. Probably too new Xcode version.
